tests/containers/helm.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    $self->{deployment_name} = "apache";
+    my $chart = "bitnami/apache";
+
+    zypper_call("in -y jq aws-cli helm podman kubernetes-client");
+
+    # Check helm version
+    assert_script_run("helm version");
+    record_info('Helm version', script_output("helm version"));
+
+    # Connexion to AWS
+    my $provider = $self->provider_factory(service => 'EKS');
+    $self->{provider} = $provider;
+
+    # Access to Cluster
+    record_info('Kubectl version', script_output("kubectl version"));
+    assert_script_run("cat \$HOME/.kube/config");
+    assert_script_run("kubectl get nodes");
+
+    # Ensure is deleted
```

Comment:
just curious, why deleting something that hasn't been created before?

Reply:
If the previous step of deletion has failed then this test never will start and manually we'll have to clean up the k8s. 
But there is an other solution, and that is to set up a new name every time, this allows to have some tests in parallel. Changing that

Reply:
This is dangerous, what if 2 tests run at the same time?

Reply:
now, no problem

Reply:
if we are generating an unique deployment name, do we still need this?